assignment1/kthInLinkedList.py

```diff
@@ -0,0 +1,40 @@
+class Node:
+    """A class that represents the node of the singly linked list.
+
+    Fields:
+        key: an integer, key of the node;
+        next: a Node, the next element in the linked list.
+    """
+    def __init__(self, _key = 0, _next = None):
+        self.key = _key
+        self.next = _next
+
+    def __str__(self):
+        return str(self.key)
+
+def kthInLinkedList(linkedList, k):
```

Comment:
The name of the function does seem misleading. Call it `kthToLastInLinkedList` instead.